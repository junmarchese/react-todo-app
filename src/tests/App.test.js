import React from "react";
import { render, screen } from "@testing-library/react";
import App from "../App";

it("renders without crashing", () => {
    render(<App />);
});

it("matches snapshot", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});

it("renders the heading", () => {
    render(<App />);
    const heading = screen.getByText("Todo App");
    expect(heading).toBeInTheDocument();
});

