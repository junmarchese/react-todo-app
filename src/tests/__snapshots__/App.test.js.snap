// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Todo App
    </h1>
    <div>
      <form>
        <label
          for="task"
        >
          <strong>
            New Todo:
          </strong>
        </label>
        <input
          id="task"
          name="task"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
      <ul />
    </div>
  </div>
</DocumentFragment>
`;
